### PR TITLE
Removed configmap envs

### PR DESCRIPTION
**Pull request by EdwardKaravakis**
Removed all used configmap envs. They should now come from the secrets.

### CODE REVIEW

File: helm/bigmon/charts/main/templates/configmap.yaml [hunk 1]
```diff
@@ -16,54 +16,6 @@ kind: ConfigMap
 metadata:
```
Comment (tmaeno) [inline helm/bigmon/charts/main/templates/configmap.yaml:16]: You can delete the env stuff completely from configmap.yaml and deployment.yaml as
proxies are set at

https://github.com/PanDAWMS/panda-k8s/blob/main/secrets/templates/bigmon.yaml#L66

Comment (tmaeno) [reply helm/bigmon/charts/main/templates/configmap.yaml]: You need to remove

https://github.com/PanDAWMS/panda-k8s/blob/main/helm/bigmon/charts/main/templates/deployment.yaml#L101-L102

BTW, please submit pull requests once you test your changes on your folks. The master branch should be functional.

Comment (EdwardKaravakis) [reply helm/bigmon/charts/main/templates/configmap.yaml]: Oh yes, I was looking for the proxy setting. Yes I have converted it into WIP.

**Comment by tmaeno** [comment]
BTW, I cherry-picked your changes into my folk for BNL OKD and merged a pull request

https://github.com/PanDAWMS/panda-k8s/pull/35

It would be good to synk your folk with the main branch before testing your changes at CERN/SLAC so that you can avoid re-discover the issues I already fixed.

**Comment by EdwardKaravakis** [comment]
Yes I just realized that you've cherry-picked some things - I was testing my changes as we speak. In any case, I will first sync and start the testing part from scratch. BTW,  I have a question for the removal of SOCIAL_AUTH_INDIGOIAMOIDC_BASEPATH , could you please check my comment?

**Pull request closed without merging**